### PR TITLE
Add Buildkite agent chart

### PR DESCRIPTION
**Pull request by rimusz**

### CODE REVIEW

File: stable/buildkite/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  kubectl --namespace={{ .Release.Namespace }} get pods -l "app={{ template "name" . }},release={{ .Release.Name }}"
+
+{{- else }}
+
+##############################################################################
+####   ERROR: You did not provide Buildkite Agent Token.                  ####
+##############################################################################
+
+All pods do not go to the running state if the Buildkite Agent Token was not provided.
```
Comment (prydonius) [inline stable/buildkite/templates/NOTES.txt]: I think "Pods will not go to the running state..." is more clear.

File: stable/buildkite/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+description: Buildkite agent chart
```
Comment (prydonius) [inline stable/buildkite/Chart.yaml]: To remove redundancy I think "Agent for Buildkite" is a better description

**Review by prydonius — COMMENTED** [review]
code review lgtm, but it would be good if someone who uses Buildkite could give this a run

**Comment by rimusz** [comment]
@prydonius updated wording as you requested

**Pull request merged**